### PR TITLE
Refactor, style, and add icons

### PR DESCRIPTION
**Pull request by whitneyfirth**
Changes made include: 
* Created a color array and mapped over the array to create the buttons for NoteForm - cleans up our render!
* Added notebook icon to header bar as an attempt to look like the Google Keep icon... also adjusted our name to just "Keeper", that is completely up for change tomorrow just wanted to get us out of "Trapper Keeper" land 😄 Same in regards to background/font color of header bar.
* Added an edit and delete icon to our NoteCards for use to click straight from the NoteCard (no functionality yet)
* Changed NoteCard/NoteContainer media query for under 528px so that the cards take up majority of the page as Google Keep does 
* Wrapped title and list items in NoteForm in a div, added overflow scroll and fixed height so that our save/delete/color palette buttons always show
* Updated snapshots

### CODE REVIEW

File: src/containers/NoteForm/NoteForm.js [hunk 1]
```diff
@@ -158,17 +158,22 @@ export class NoteForm extends Component {
     const { title, color, showColorOptions } = this.state;
     const { status } = this.props;
     const { path } = this.props.match;
+    const colors = ['white', 'pink', 'orange', 'yellow', 'green', 'blue', 'lavender', 'gray'];
```
Comment (trbachmann) [inline src/containers/NoteForm/NoteForm.js:161]: Nice! This cleans up our render a lot and now we can easily add more colored buttons in the future if we want to give users additional options 🔥

File: src/containers/NoteForm/NoteForm.js [hunk 2]
```diff
@@ -158,17 +158,22 @@ export class NoteForm extends Component {
...
+      return <button key={color} id={color} className='NoteForm--color' onClick={this.handleColorChoice}></button >
+    });
+
     return (
       <form className={'NoteForm--background-' + color} onSubmit={this.handleSubmit}>
-        {this.getTitleInput()}
-        {this.getIncompleteListItems()}
-        {this.getNewListItemInput()}
-        {this.getCompleteListItems()}
+        <div className='NoteForm--title-and-listitems'>
```
Comment (trbachmann) [inline src/containers/NoteForm/NoteForm.js:168]: This looks really good when a user has a large number of list items, but can still see the save/delete btns and the palette picker. Nice work!

**Review by trbachmann — APPROVED** [review]
Nice work! I really like how the header turned out and the change to Keeper! At first I was concerned it would feel too empty if we removed the blue from the header, but it looks really clean 😃great job! I think when we get the welcome example card in there too, it'll compliment it well. Also glad we have the buttons refactored, if we want to change/add too the colors options in the future it'll be quick.

File: src/containers/App/App.scss [hunk 3]
```diff
@@ -3,12 +3,21 @@
...
+  font-weight: 500;
   margin: 0;
-  text-align: left;
-  padding: 24px;
-  font-weight: 400;
+  padding: 0% 4%;
+  height: 65px;
+  display: flex;
+  align-items: center;
+  justify-content: left;
```
Comment (dForDeveloper) [inline src/containers/App/App.scss:16]: This looks great. 🙂

File: src/containers/NoteCard/NoteCard.js [hunk 4]
```diff
@@ -45,6 +47,10 @@ export const NoteCard = (props) => {
     <Link to={'/notes/' + id} className={'NoteCard--background-' + color}>
       <h3 className='NoteCard--h3'>{title}</h3>
       <div>{result}</div>
+      <span className='NoteCard--icon-container'>
+        <img className='NoteCard--delete-icon' src={editicon} alt='edit icon'/>
+        <img className='NoteCard--edit-icon' src={deleteicon} alt='delete icon'/>
+      </span>
```
Comment (dForDeveloper) [inline src/containers/NoteCard/NoteCard.js:53]: Good idea. Now we can make the checkboxes clickable.

File: src/containers/NoteForm/NoteForm.js [hunk 5]
```diff
@@ -158,17 +158,22 @@ export class NoteForm extends Component {
     const { title, color, showColorOptions } = this.state;
     const { status } = this.props;
     const { path } = this.props.match;
+    const colors = ['white', 'pink', 'orange', 'yellow', 'green', 'blue', 'lavender', 'gray'];
+    const buttons = colors.map(color => {
+      return <button key={color} id={color} className='NoteForm--color' onClick={this.handleColorChoice}></button >
```
Comment (dForDeveloper) [inline src/containers/NoteForm/NoteForm.js:163]: The return is so much cleaner now. 👍

File: src/containers/NoteForm/NoteForm.scss [hunk 6]
```diff
@@ -3,15 +3,18 @@
...
   top: 96px;
   border-radius: 10px;
   padding: 16px;
-  overflow-y: auto;
   box-shadow: $heavy-shadow;
 }
 
+.NoteForm--title-and-listitems {
+  overflow: scroll;
+  max-height: 350px;
```
Comment (dForDeveloper) [inline src/containers/NoteForm/NoteForm.scss:15]: This is awesome.

**Review by dForDeveloper — APPROVED** [review]

**Pull request merged**